### PR TITLE
fix: don't eslint on js_binary, it has no srcs

### PR DESCRIPTION
**Pull request by alexeagle**

### CODE REVIEW

**Review by jbedard — APPROVED** [review]

**Comment by jbedard** [comment]
Are tests easy enough to write for a simple edge case like this?

File: lint/eslint.bzl [hunk 1]
```diff
@@ -88,6 +88,8 @@ def eslint_action(ctx, executable, srcs, report, use_exit_code = False):
 def _eslint_aspect_impl(target, ctx):
     if ctx.rule.kind not in ["js_binary", "js_library", "ts_project", "ts_project_rule"]:
         return []
+    if not hasattr(ctx.rule.files, "srcs"):
```
[the remark this answers is not in the document]
Comment (alexeagle) [reply lint/eslint.bzl:91]: oh, in testing that question I found this was a different bug - https://docs.aspect.build/rulesets/aspect_rules_js/docs/js_binary doesn't allow `srcs` at all. I'll just remove `js_binary` from this list for now.
